portmap/core/views.py

```diff
@@ -216,3 +220,34 @@ def debug_list_articles(request):
 
 def debug_help_dev(request):
     return TemplateResponse(request, "core/debug_index.html")
+
+class CorrectMimeTypeFeed(DefaultFeed):
+    content_type = 'application/xml; charset=utf-8'
+
+class RssFeed(Feed):
+    feed_type = CorrectMimeTypeFeed
+    title_template = "feed/article_title.html"
+    description_template = "feed/article_description.html"
+    title = "RSS Feed"
+    link = "https://portmap.dtinit.org/articles_feed"
+    portmap_link = "https://portmap.dtinit.org"
+    description = "Articles from Portability Articles repo"
+
+    def items(self):
+        parsed_articles = []
+
+        for article in Article.objects.all():
```

Comment:
Should we have some sort of deterministic ordering for articles? I think most article feeds are ordered most recent to least, but we don't track publish dates. I have no idea whether this actually matters!

Reply:
The default ordering is almost certainly by id (sequence generated internally by django) so it will effectively order consistently by recency.  We should flip that and change to date, which means that we should add created_at to the Article data model.  That will probably end up being useful anyway.  We can do this in an update though.